cmd/init.go

```diff
@@ -20,10 +20,24 @@ THE SOFTWARE.
 package cmd
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+
 	"github.com/facebookincubator/ttpforge/pkg/logging"
+	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 )
 
+func copyEmbeddedConfigToPath(configFilePath string) error {
```

Comment:
Now. you must explicitly run `init` if you want the embedded config to get copied - this makes TTPForge compatible with deployment scenarios in which you don't want a `config.yaml`